### PR TITLE
Job-editor prototype

### PR DESCRIPTION
**Pull request by hao1939**
1. Launcher a Jupyter server
2. Hosted a mnist pytorch example
3. Scripts for automatically submitting the example to cluster

image
image

### CODE REVIEW

**Comment by Gerhut** [comment]
<http://10.151.40.179/a/10.151.40.176:31464/lab> (through pylon) should works, but it seems some additional works needed.

**Comment by hao1939** [comment]
@Gerhut ,
Yes, it relates to jupyter base url.

One question, I'm not sure whether we should using http://10.151.40.179/a/10.151.40.176:31464/lab.
Are we going to publicly announce the pylon proxy feature?

**Comment by fanyangCS** [comment]
We already have an example Jupyter: https://github.com/Microsoft/pai/tree/master/examples/jupyter. Please explain in the doc what difference this example is from the existing Juypter example.

**Comment by hao1939** [comment]
@fanyangCS ,
Great, I updated the readme.
The purpose of this example is demonstrating how we leverage PAI restserver APIs to create an online job editor.

**Comment by fanyangCS** [comment]
I suggest you refine the first paragraph of readme. I still don't quite understand what is a job editor for? are you saying users can run the job, edit the code in the job container, and test it on the fly, thus this enables interactive debugging?

**Comment by hao1939** [comment]
Yeah, that's right. I will update it.

File: examples/job-editor/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+# Launcher a jupyter job-editor for interactive debugging
+
+The example is a prototype of online job editor. It demonstrates:
+
+1. Launcher a jupyter notebook for online editing and interactive debugging. The editor is running on PAI, so user will debug on the same environment as the job will be running and get the feedback on the fly.
```
Comment (fanyangCS) [inline examples/job-editor/README.md]: ...for online *code* editing ...

**Pull request merged**